internal/worker/remoterelationconsumer/localconsumerworker.go

```diff
@@ -259,10 +259,10 @@ func (w *localConsumerWorker) loop() (err error) {
 		return errors.Trace(err)
 	}
 
-	offerStatusWatcher, err := w.watchRemoteOfferStatus(ctx)
-	if err != nil {
-		return errors.Trace(err)
-	}
+	// offerStatusWatcher, err := w.watchRemoteOfferStatus(ctx)
+	// if err != nil {
+	// 	return errors.Trace(err)
+	// }
```

Comment:
@jack-w-shaw has a follow up PR to make this work.

Reply:
Yeah sorry I committed the wrong file :rofl: